apps/judicial-system/web/src/components/FormProvider/limitedAccessCase.graphql

```diff
@@ -54,6 +54,7 @@ query LimitedAccessCase($input: CaseQueryInput!) {
       id
       name
       title
+      email
```

Comment:
Isn't the defender also allowed to see the email addresses of the judge and registrar? If so please add that to the field selectors below

Reply:
Jess! Thanks 🙏